.gitignore

```diff
@@ -13,3 +13,5 @@ target-bin/bootstrap-fixup
 .vagrant
 docker
 *.Dockerfile
+cache
+target-*
```

Comment:
ACK `cache`, why `target-*` ?

Reply:
@fanquake 
It creates (particularly using LXC) two files:
https://github.com/devrandom/gitian-builder/blob/4053f3c88e6d16d174b6983f01d5976958576ab5/bin/make-base-vm#L117 and
https://github.com/devrandom/gitian-builder/blob/3934150c5a0715212ff8794612c222aae406f28d/libexec/make-clean-vm#L57 E.g., `base-bionic-amd64` and `target-bionic-amd64`.

The former is ignored already:
https://github.com/devrandom/gitian-builder/blob/27d01300d2d312d31f4f1cfbf872db9e891056a1/.gitignore#L9

The latter (`target-*` or `target*`) should be ignored too.